build.gradle

```diff
@@ -225,12 +228,16 @@ subprojects { subproject ->
 
         compile group: 'javax.inject', name: 'javax.inject', version: '1'
 
-        compile group: 'com.microsoft.azure', name: 'azure-storage', version: '8.0.0'
+        compile (group: 'com.microsoft.azure', name: 'azure-storage', version: '8.0.0') {
+            exclude group: 'com.google.guava', module: 'guava'
+        }
 
         // To avoid compiler warnings about @API annotations in JUnit5 code.
         testCompileOnly 'org.apiguardian:apiguardian-api:1.0.0'
-
-        testCompile group: 'com.opentable.components', name: 'otj-pg-embedded', version: '0.12.0'
+        testCompile "org.postgresql:postgresql:42.2.16"
+        testCompile (group: 'com.opentable.components', name: 'otj-pg-embedded', version: '0.13.3') {
+            exclude group: 'org.postgresql', module: 'postgresql'
```

Comment:
can we add global exclude instead of doing in individual file?
Something like below:
   configurations.all {
        collect { configuration ->
            configuration.exclude group: 'org.apache.logging.log4j', module: 'log4j-to-slf4j'
        }

Reply:
We still need one of the version of these package, global exclude would not work.